demojustice.yaml

```diff
@@ -592,7 +603,34 @@ associationDefinitions:
         transforms:
         - !<transforms.ConcatTransform>
           columns: ["SubjectIdentification", "ChargeID"]
-    name: "chargedwith"
+      general.datetime:
+        type: "general.datetime"
+        column: "ArrestDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["MM/dd/yy", "M/dd/yy", "MM/d/yy", "M/d/yy", "MM/dd/yyyy", "M/dd/yyyy", "MM/d/yyyy", "M/d/yyyy"]
+          timezone: "America/Los_Angeles"
+    name: "chargedwith1"
+
+  chargedwith2:
+    fqn: "ol.chargedwith"
+    entitySetName: "DemoChargedWith"
+    src: "person"
+    dst: "charge"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["SubjectIdentification", "ChargeID"]
+      general.datetime:
+        type: "general.datetime"
+        column: "ArrestDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["MM/dd/yy", "M/dd/yy", "MM/d/yy", "M/d/yy", "MM/dd/yyyy", "M/dd/yyyy", "MM/d/yyyy", "M/d/yyyy"]
+          timezone: "America/Los_Angeles"
+    name: "chargedwith2"
 
   chargedwith2:
```

Comment:
duplicate key

Reply:
odd, why didn't that throw an error in shuttle...